Components/Buttons.html

```diff
@@ -0,0 +1,176 @@
+        <ion-icon name="menu-outline"></ion-icon>
+      </div>
+    </nav>
+
+    <!-- MAIN CONTAINER -->
+
+    <div class="mainCont">
+      <!-- SIDEBAR CONTAINER -->
+
+      <div class="sidebarCont hideSidebar">
+        <div class="sidebarList">
+          <a class="sidebarListItem" href="docs.html">Getting Started</a>
+          <hr class="hr-dark mt8" />
+          <a class="sidebarListItem" href="./Typography.html">Typography</a>
+          <a class="sidebarListItem selected" href="./Buttons.html">Buttons</a>
+          <a class="sidebarListItem" href="./Cards.html">Cards</a>
+          <a class="sidebarListItem" href="./Images.html">Images</a>
+          <a class="sidebarListItem" href="./inputs.html">Inputs</a>
+          <a class="sidebarListItem" href="./lists.html">Lists</a>
+          <a class="sidebarListItem" href="./Navigation.html">Navigation</a>
+          <a class="sidebarListItem" href="./Avatars.html">Avatars</a>
+          <a class="sidebarListItem" href="./Alerts.html">Alerts</a>
+          <a class="sidebarListItem" href="./miscellaneous.html"
+            >Miscellaneous</a
+          >
+        </div>
+        <div class="signature-cont">
+          <hr class="hr-dark mt8" />
+          <h4 class="signature-text">
+            Made with 🤎 by
+            <a class="signature-link" href="https://atishayjain.netlify.app/"
+              >Atishay Jain</a
+            >
+          </h4>
+        </div>
+      </div>
+
+      <!-- MAIN CONTENT CONTAINER -->
+
+      <div class="contentCont">
+        <h1 class="heading-l">Buttons</h1>
+        <p class="para-m mt4">
+          Different types of buttons for a better user Experience
+        </p>
+        <hr class="hr-light mt8" />
+
+        <h4 class="heading-s mt10">Examples:</h4>
+
+        <!-- PRIMARY BUTTONS AND SHAPE -->
+
+        <div class="exampleCont">
+          <h3 class="exampleText">Default Buttons in three shapes</h3>
+          <div class="flexCont">
+            <button class="btn primary-btn btn-pill">Button</button>
+            <button class="btn secondry-btn btn-square">Button</button>
+            <button class="btn disabled-btn btn-rounded">Button</button>
+          </div>
+
+          <h3 class="showCodeToggle" onclick="showCodeToggle()"></h3>
+          <div class="code hideCode">
+            <iframe
+              src="https://carbon.now.sh/embed/gNoXnB8KrMfQNuSeZvac"
+              style="
+                width: 100%;
+                height: 286px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+        </div>
+
+        <!-- LINK BUTTONS -->
+
+        <div class="exampleCont">
+          <h3 class="exampleText">Link buttons</h3>
+          <div class="flexCont">
+            <button class="btn primary-btn btn-pill">
```

Comment:
inconsistent naming of button classes,be consistent with a single name style,would suggest go with btn-primary,btn-secondary and so on

Reply:
changed naming as btn-primary